### ISSUE TITLE
Add an xtask task to update CI binaries

### ISSUE THREAD

**Issue by Luni-4** [tracel-ai/xtask#8]
### Feature description

Add an `xtask` task to automatically check for whether there are new versions of Continuous Integration binaries. It might be structured in the following way:

1. Use [serde_yaml](https://crates.io/crates/serde_yaml) to parse the `yml` files in search of environment variables which could be defined as:
   - `_BINARY_URL: "url"`
   - `_BINARY_NAME_VERSION: "x.x.x"`
3. Using `url` to evaluate if there is a `BINARY_NAME` with a release version greater than `x.x.x` through the [reqwest](https://crates.io/crates/reqwest) crate
4. If there is a new version, the task updates the `_BINARY_NAME_VERSION` value, thus changing files

### Feature motivation

This task makes less tedious the updating of Continuous Integration binaries. This check can also be integrated into a Continuous Integration action which automatically opens a PR with the updates.

**Comment by syl20bnr** [comment]
Yes you are right, transferred to [tracel-ai/xtask](https://github.com/tracel-ai/xtask) repo.
I believe we should do 1 and have all the actions or workflows depends on `tracel-xtask` but we need a command that is flexible enough to work everywhere.

**Comment by Luni-4** [comment]
Just looking at this issue @syl20bnr, and I do not know if it's still valid, as there have been many changes across the entire `burn` infrastructure. If I'm not wrong, most of the updates can be done automatically through `dependabot` in `weekly` mode.
